src/components/LetterInputs.js

```diff
@@ -0,0 +1,53 @@
+import { useState } from "react";
+import { useDispatch } from "react-redux";
+import { update } from "../reducers/word";
+import { increment } from "../reducers/Increment";
+import { useSelector } from "react-redux";
+
+function LetterInputs({ wordLength }) {
+  const [letterStates, setLetterStates] = useState(Array(wordLength).fill(""));
```

Comment:
I'll keep looking, is it necessary to set the initial value to something so specific than an empty array?